adsft/tests/test_extraction.py

```diff
@@ -311,11 +468,10 @@ def test_that_we_can_parse_the_xml_content(self):
         """
 
         full_text_content = self.extractor.open_xml()
-        for parser_name in ("html5lib", "html.parser", "lxml-html", "direct-lxml-html", "lxml-xml", "direct-lxml-xml",):
-            self.extractor.parse_xml(preferred_parser_names=(parser_name,))
-            journal_title = self.extractor.extract_string('//title')
+        self.extractor.parse_xml()
```

Comment:
We are not going through the whole list of parsers here (in this whole class) while we were doing it before.